src/api-functions/get-issues-by-project.ts

```diff
@@ -0,0 +1,101 @@
+import dotenv from "dotenv";
+
+export interface Issue {
```

Comment:
Extract these object interface inside a model directory and other files. Dont mess them around with the implementations.